### DISCUSSION TITLE
I am a PhD student so of course i am not in the compus, I am writing my thesis and i do not use student card for god sake.

### DISCUSSION THREAD

**Discussion by sagirogluata** [community/community#133563] [GitHub Education]
### Hi everyone,

Well i am a PhDc I do not go university, we do all the things online and there is no card or "reasoning paper" for that. I think the ai approval is such a funny thing. And in my country you can get the qr coded paper from goverment's official website. And you didn't approve it too. So what do i have to do? to go to the campus which is probably 500 km or more away from me now and go to library and get a new card which i never use because i do not go to the university for any kind of reason because I am a phdc and we never go university. so how can i approve you guys i am a phdc. I can send you the academic papers i wrote which is published. or what else? ORCID id number? my accademia account? Research gate account? what do you want?

**Comment by InboraStudio** [comment]
uhmmm so Yeah ! It sounds like you're trying to verify your PhD candidate status for some sort of approval process, perhaps related to software or service access that requires proof of academic standing. Since you mentioned various forms of proof, here's a list of documents and steps that might be accepted by many institutions or services for such verification:

## Here same way mey it Help you sooo Yeah here we Goooooo

ORCID iD: This is a unique identifier for researchers and can serve as proof of your academic contributions and affiliations. You can share your ORCID iD and provide a link to your profile.

Published Academic Papers: Provide links or PDF copies of your published papers. These should include your name, the name of your institution, and publication details.

Official Email: Use your official university email address to correspond with the service or institution. Emails from official academic domains (e.g., .edu, .ac, etc.) are often accepted as proof of affiliation.

Letter of Enrollment or Proof of Candidacy: Request a formal letter from your academic advisor or department head that confirms your status as a PhD candidate. Even if you don't visit the campus, this can often be done via email.

ResearchGate or Academia.edu Profile: Provide links to your profiles on academic networking sites where your affiliation and research work are listed.

University Portal: If your university has an online portal where your status is listed, you can provide a screenshot of this portal showing your enrollment and status.

QR Coded Paper from Government Website: If your government provides a QR coded document verifying your academic status, provide this along with instructions on how to verify it.

Supervisor's Confirmation: An email or letter from your supervisor confirming your status and affiliation might be accepted.

## Here’s how you can present this information to the service or institution for verification:

## You can make a email by like this mate see am giveing you a code

```vbnet
Subject: Verification of PhD Candidate Status

Dear [Recipient's Name],

I am writing to provide the necessary documents to verify my status as a PhD candidate at [University Name]. Due to the nature of my program, I am primarily engaged in online and remote work, and do not have a physical university card. However, I can provide the following documents as proof of my academic status:

1. **ORCID iD**: [Link to ORCID profile]
2. **Published Papers**: [Links to academic papers]
3. **Official Email**: [Your university email address]
4. **Letter from Advisor**: Attached is a letter from my academic advisor confirming my status.
5. **ResearchGate Profile**: [Link to ResearchGate profile]
6. **QR Coded Paper**: Attached is the QR coded document provided by my government, which verifies my academic status.

I hope this information suffices for the verification process. Please let me know if there are any additional documents or steps required.

Thank you for your understanding and assistance.

Sincerely,
[Your Name]
PhD Candidate, [Department Name]
[University Name]
[Contact Information]
````

Attachments
PDF copies of your published papers.
Letter from your advisor.
QR coded document.
Screenshots of any relevant university portals.

**Reply by samus-aran** [reply]
That is a very interesting "tone of voice" prompt 👀

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

However, I do want to pull out some interesting points here:

- Students/Educators can provide additional evidence not just Student ID or Academic Email
- We **do not do manual reviews** so please don't reach out to Support if it's to provide supplement evidence.

**Comment by sagirogluata** [comment]
Oh yea! to which e mail adress?

**Comment by samus-aran** [comment]
Hi @sagirogluata 👋 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. 

- **Ensure you have updated your Billing info, this is essential.**

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍

**Comment by sagirogluata** [comment]
I am writing to provide the necessary documents to verify my status as
a PhD candidate at [University Name]. Due to the nature of my program,
I am primarily engaged in online and remote work, and do not have a
physical university card. However, I can provide the following
documents as proof of my academic status:
1. **ORCID iD**: Ata Sağıroğlu (0000-0003-4707-4771) - My ORCID
<https://orcid.org/my-orcid?orcid=0000-0003-4707-4771>2. **Published
Papers**: (99+) Ata Sağıroğlu | Dokuz Eylül University - Academia.edu
<https://deu.academia.edu/atasagiroglu>3. **Official Email**:
***@***.***
4. **ResearchGate Profile**: Ata SAĞIROĞLU | Bachelor of Arts | Dokuz
Eylul University, İzmir | DEÜ | The Institute of Fine Arts | Research
profile (researchgate.net)
<https://www.researchgate.net/profile/Ata-Sagiroglu>6. **QR Coded
Paper**: Attached is the QR coded document provided by my government,
which verifies my academic status.
I hope this information suffices for the verification process. Please
let me know if there are any additional documents or steps required.
Thank you for your understanding and assistance.
Sincerely,
Ata SağrıoğluPhD Candidate, Musicology
Dokuz Eylul ***@***.***

Attachments
[image: Screenshots of any relevant university portals.png]

R coded document.
Screenshots of any relevant university portals.
